integration-test/src/lib.rs

```diff
@@ -36,19 +38,27 @@ use enso_gui::initializer::setup_global_executor;
 pub struct IntegrationTest {
     pub executor: EventLoopExecutor,
     pub ide:      Ide,
+    pub root_div: HtmlDivElement,
 }
 
 impl IntegrationTest {
     /// Initializes the executor and `Ide` structure and returns new Fixture
     pub async fn setup() -> Self {
         enso_web::forward_panic_hook_to_error();
         let executor = setup_global_executor();
-        let div = enso_web::create_div();
-        div.set_id("root");
-        enso_web::body().append_with_node_1(&div).expect("Failed to add root div element.");
+        let root_div = enso_web::create_div();
+        root_div.set_id("root");
+        root_div.set_style_or_panic("display", "none");
+        enso_web::body().append_with_node_1(&root_div).expect("Failed to add root div element.");
```

Comment:
I think you should use `append_or_panic` from `NodeInserter` trait instead of web-sys stuff.